functional/FunctionalTest.java

```diff
@@ -582,7 +582,18 @@ public static void setup() throws Exception {
 
   /** Tear down test setup. */
   public static void teardown() throws Exception {
-    client.removeBucket(RemoveBucketArgs.builder().bucket(bucketName).build());
+    String methodName = "removeBucket(RemoveBucketArgs args)";
+
+    if (!mintEnv) {
+      System.out.println("Test: " + methodName);
```

Comment:
This is not a test. It is a clean up function of `setup()`. If `mint` is the goal for this PR, just remove this.